templates/webapi/admin/job_template/index.html.ep

```diff
@@ -161,6 +161,24 @@ scenarios:
     <i>medium</i>-<b>x86_64</b>:
       <b>*tests</b>
                         </code>
+                        <div>Re-use settings with YAML merge keys:</div>
+<code>scenarios:
+  <b>i586</b>:
+    <i>medium</i>-<b>i586</b>:
+    - foo
+    - bar:
+        machine: <b>64bit</b>
+        priority: <b>70</b>
+        settings: <b>&amp;common</b>
+          FOO: <b>a</b>
+          BAR: <b>b</b>
+    - baz:
+        machine: <b>64bit</b>
+        priority: <b>70</b>
+        settings:
+          <<: <b>*common</b>
```

Comment:
It might be better to escape the `<<`.

Reply:
How should I escape it? With backticks? The block is already in backticks.

Reply:
https://stackoverflow.com/questions/7381974/which-characters-need-to-be-escaped-in-html#7382028

Reply:
Oh sorry, I thought this was in the asciidoc example...

Reply:
thanks, done!